doc/tutorial/08_tls_enforced.md

```diff
@@ -0,0 +1,91 @@
+## Preface 
+
+This tutorial will take you through the connection path from client to `pgagroal` to `postgres` server when TLS is enforced.
+
+## Setup
+
+To enforce tls along the whole path, we first need to create X509 certicates for client->pgagroal and pgagroal->postgres seperately.
```

Comment:
tls -> TLS, client->pgagroal -> client to pgagroal, pgagroal->postgres -> pgagroal to PostgreSQL